runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -53,393 +58,167 @@ Icons appear at the top of each page in the Runtime Manager documentation, indic
+* <<High Availability>>
+* <<JDK Versions>>
+* <<Load Balancing>>
+* <<Logging>>
+* <<Monitoring>>
+* <<Object Store>>
+* <<Scheduling>>
+* <<Security Updates>>
+* <<Shared Resource Support>>
+* <<Troubleshooting>>
 
 
+=== High Availability
 
-== CloudHub Deployments
+[%header,cols="15,50,20"]
+|===
+|Environment | Implementation      | Reference
+|CloudHub 2.0 | Via clustering with two or more replicas, and Anypoint VPN. |   xref:cloudhub-2::ch2-clustering.adoc[Clustering], xref:cloudhub-2::ps-vpn-about.adoc#vpn-ha[VPN High Availability]. 
+|CloudHub 1.0 |Via clustering, with two or more workers, and Anypoint VPN. | xref:cloudhub-fabric.adoc[Clustering], xref:vpn-high-availability.adoc[Anypoint VPN High Availability].
+|Hybrid | Via server groups or clusters. Clustered Mule instances have distributed shared memory, which provides persistent VM queues, transactions, and cluster-wide data storage. | xref:server-group-about.adoc[Server Groups], xref:cluster-about.adoc[Clusters], and xref:mule-runtime::mule-high-availability-ha-clusters.adoc[Mule Runtime High Availability (HA) Cluster Overview].
+|Private Cloud Edition | Via server groups or clusters. Clustered Mule instances have distributed shared memory, which provides persistent VM queues, transactions, and cluster-wide data storage. | xref:server-group-about.adoc[Server Groups], xref:cluster-about.adoc[Clusters], and xref:mule-runtime::mule-high-availability-ha-clusters.adoc[Mule Runtime High Availability (HA) Cluster Overview].
+|Runtime Fabric | Via automatic application fail-over. You obtain high availability when you deploy two or more replicas per application. |  xref:runtime-fabric::deploy-resource-allocation.adoc[Deploy a Mule Application to Runtime Fabric].
+|===
 
+=== JDK Versions
 
-image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
-image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
-image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
-image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+[%header,cols="15,45,45"]
+|===
+|Environment | Version         | Reference
+|CloudHub 2.0 | JDK 11 | xref:release-notes::cloudhub-2/cloudhub-2-runtimes-release-notes.adoc[CloudHub 2.0 Runtime Release Notes].
+|CloudHub 1.0 |JDK 11 | xref:release-notes::cloudhub/cloudhub-runtimes-release-notes.adoc[CloudHub Runtime and Operating System Release Notes].
+|Hybrid | See the  Mule runtime engine release notes to determine the minimum JDK supported version. | xref:release-notes::mule-runtime/mule-esb.adoc[Mule Runtime Release Notes].
+|Private Cloud Edition | See the  Mule runtime engine release notes to determine the minimum JDK supported version. |xref:release-notes::mule-runtime/mule-esb.adoc[Mule Runtime Release Notes].
```

Comment:
See line 98.

Reply:
For these, if you don't add the text in between the brackets, it will get picked up automatically and you don't need to keep them updated. 
The CH2 RN title is actually "CloudHub 2.0 Runtime and Infrastructure Patch Update Release Notes".

Reply:
Applied